handbook/engineering/search/index.md

```diff
@@ -47,15 +50,20 @@ Our current focus is documented in the [tracking issue for the current milestone
   - **Outcomes:** OpenGrok users eagerly migrate to Sourcegraph, and can run complex searches on Sourcegraph.
   - **Owner:** Rijnard
   - **Status:** In progress
+  - **Estimated completion:** TODO
 - **Improve syntax for existing filters like `repohasfile`**
   - **Problem:** Creating a custom filter name for every permutation of search use does not scale (existing filter examples: `repohasfile`, `hascommitafter`).
   - **Outcomes:** We have an expressive syntax that scales and effectively leverages existing filters to achieve searches that previously would have required a custom/new filter.
+  - **Tentative Plan:** Implement a query language extension to express relations (i.e., rules) on code, files, repositories, and commits
   - **Owner:** Rijnard/Stefan
   - **Status:** Not started
+  - **Estimated start:** TODO
```

Comment:
@rvantonder @stefanhengl could you add an (even rough) estimated start / estimated effort for this work? Eg. `Estimated start: 3.22, estimated effort: 3 months`